suite-native/atoms/src/Alert.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Icon } from '@trezor/icons';
+
+import { Box } from './Box';
+import { Text } from './Text';
+
+type AlertProps = {
+    title: string;
+};
+
+const alertWrapperStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    alignItems: 'center',
+    padding: utils.spacings.medium,
+    borderRadius: utils.borders.radii.medium,
+    backgroundColor: utils.colors.backgroundAlertBlueSubtleOnElevation0,
+}));
+
+const textWidthStyle = prepareNativeStyle(_ => ({
+    width: '80%',
+    marginLeft: 12,
+}));
+
+const ICON_SIZE = 48;
+const iconWrapperStyle = prepareNativeStyle(utils => ({
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: ICON_SIZE,
+    height: ICON_SIZE,
+    backgroundColor: utils.colors.backgroundAlertBlueSubtleOnElevation1,
+    borderRadius: utils.borders.radii.round,
+}));
+
+export const Alert = ({ title }: AlertProps) => {
```

Comment:
That one is AlertSheet though. I think Alert is proper naming for this component and since it can be reused I would leave it in atoms. 

I was thinking the same but I guess since it's distinguished by the Sheet suffix in the other component, it's fine by me. The alert sheet is not actually alert by definition and this one is if we really think about it. 

Could move it to onboarding package, but why not make this reusable if we need it in the future?

Reply:
Agree with @PeKne that Alert is not good name. At least go with something like `InfoBox` or `AlertBox`